cargo-shuttle/Cargo.toml

```diff
@@ -54,6 +54,8 @@ url = "2.3.1"
 uuid = { workspace = true, features = ["v4"] }
 webbrowser = "0.8.2"
 semver = "1.0.17"
+aws-sdk-dynamodb = { workspace = true }
+aws-config = { workspace = true }
```

Comment:
We should confirm these dependencies are needed in the CLI. I think only the tests use them :man_shrugging: